hack/setup-rbac/main.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/openshift/openshift-azure/pkg/util/azureclient/authorization"
 )
 
+const (
+	OSAContributorAddon = "9bef4004-f578-4ae8-8364-698f59f1bedc"
```

Comment:
What does this ID refer to?

Reply:
It refers to customer role we have to grant to us to be able to do rotations. But im struggling to craft access model for this :/